components/brave_ads/browser/features.cc

```diff
@@ -0,0 +1,149 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/browser/features.h"
+
+#include "base/feature_list.h"
+#include "build/build_config.h"
+
+namespace brave_ads {
+namespace features {
+
+const base::Feature kAdNotifications{"AdNotifications",
+                                     base::FEATURE_ENABLED_BY_DEFAULT};
+
+namespace {
+
+// Set to true to show custom ad notifications or false to show system
+// notifications
+const char kFieldTrialParameterShouldShowCustomAdNotifications[] =
+    "should_show_custom_notifications";
+const int kDefaultShouldShowCustomAdNotifications = true;
+
+// Ad notification fade animation duration in milliseconds
+const char kFieldTrialParameterAdNotificationFadeDuration[] =
+    "ad_notification_fade_duration";
+const int kDefaultAdNotificationFadeDuration = 200;
+
+// Ad notification timeout in seconds. Set to 0 to never time out
+const char kFieldTrialParameterAdNotificationTimeout[] =
+    "ad_notification_timeout";
+#if !defined(OS_ANDROID)
+const int kDefaultAdNotificationTimeout = 120;
+#else
+const int kDefaultAdNotificationTimeout = 30;
+#endif
+
+// Ad notification normalized display coordinate for the x component should be
+// between 0.0 and 1.0; coordinates outside this range will be adjusted to fit
+// the work area. Set to 0.0 for left, 0.5 for center or 1.0 for right
+const char kFieldTrialParameterAdNotificationNormalizedDisplayCoordinateX[] =
+    "ad_notification_normalized_display_coordinate_x";
+#if defined(OS_WIN)
+const double kDefaultAdNotificationNormalizedDisplayCoordinateX = 1.0;
+#elif defined(OS_MAC)
+const double kDefaultAdNotificationNormalizedDisplayCoordinateX = 1.0;
+#elif defined(OS_LINUX)
+const double kDefaultAdNotificationNormalizedDisplayCoordinateX = 1.0;
+#elif defined(OS_ANDROID)
+const double kDefaultAdNotificationNormalizedDisplayCoordinateX = 0.5;
```

Comment:
Is this parameter used on Android? If not, would you put a comment mentioning that this parameter isn't used.

Reply:
Refactored to remove unused code on Android